### ISSUE TITLE
Use the `atom-workspace` tag instead of the `workspace` class.

### ISSUE THREAD

**Issue by tophattom** [yosuke-furukawa/language-jsx#12]
In keymaps/jsx.cson: Use the `atom-workspace` tag instead of the `workspace` class.

**Comment by yosuke-furukawa** [comment]
I have not published this library. thanks.

**Comment by iamstarkov** [comment]
@yosuke-furukawa Will you update the package?

**Comment by yosuke-furukawa** [comment]
@iamstarkov 
I already removed this library from apm. Please tell me the reason why I update the package.

**Comment by iamstarkov** [comment]
I don’t know, mainly because I still have this plugin in my atom and it doesn’t tell me that you removed it from apm

**Comment by yosuke-furukawa** [comment]
@iamphill Could you try to remove my plugin ?? 

```
$ apm uninstall language-jsx
```

**Comment by iamstarkov** [comment]
I’ll do it. Maybe we are suppoused to open issue for atom about this strange behaviour?

**Comment by iamstarkov** [comment]
@yosuke-furukawa have you already created discussed issue?

**Comment by yosuke-furukawa** [comment]
no. I think the `language-jsx` users have to uninstall manually. The behavior is same for `npm` .
I still open this issue for describing uninstall process.

```
$ apm uninstall language-jsx
```
